docs/ide4c/vscode/tasks.json

```diff
@@ -0,0 +1,41 @@
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "Makefile", // You can choose your own label.
+            "type": "shell",
+            "command": "make",
+            // Make this the default build command.
+            "group": {
+                "kind": "build",
+                "isDefault": true
+            },
+            "presentation": {
+                "reveal": "always"
+            },
+            // You can pass all the building arguments here.
+            "args": [
+                "all",
+                "--debug", // Enable debug mode.
+                "-j8" // Use 8 cores.
+            ]
+        },
+        {
+            "label": "buildTest", // for building test cases before running it.
+            "command": "make", // Could be any other shell command
+            "args": [
+                "-j16",
```

Comment:
`-j16` is a bit much, no?  I think Andy found that `-j6` gave the best performance and anything more than `-j6` had rapidly diminishing returns.

Reply:
Okay, I will make it -j6. However, we could not proof it is the best. For example, using Core i9 process may differ. Therefore, I will leave a comment states that is just the number of cores.